website/docs/getting-started/contribution/index.md

```diff
@@ -1,4 +1,26 @@
-## Contributing
+---
+title: Contribution
+---
+
+## Overview
+
+While we don't accept direct contributions of new components or patterns, we do welcome contributions in the following forms:
+
+* [Reporting bugs](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform) and [giving feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
+* [Proposing new components or patterns](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform)
+* Improving our documentation
+* Participating in discussions or coming to Office Hours
+
+We continue to listen to feedback as we consider the best way to open contributions further.
+
+## Code Contributions 
+
+For contributions to the individual code packages that make up the design system, please consult the `CONTRIBUTING.md` files in each package.
```

Comment:
I would change the order (tokens, icons, components)

Also, I would add a follow-up ticket post launch, to review this content (as is now, is pretty useless for a potential "contributor" and seems to contradict what stated above that we don't accept contributions on components

Reply:
> seems to contradict what stated above that we don't accept contributions on components

we dont't accept new components, but those files include instructions for how to work on the projects if they wanted to contribute bug fixes. in either case, we do need to flesh this out more in time.

Reply:
updated the order

Reply:
@didoo @Dhaulagiri What about adding some language before "For contributions to the individual code packages" around , something like: 

"We also accept minor contributions to existing code packages, such as fixing a bug, proposing a code refactor, etc. For contributions to the individual code packages..."